src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -715,33 +717,69 @@ void ComputeTimeDerivative<Metavariables>::volume_terms(
+              *box));
+    } else {
+      ERROR("Unsupported DG formulation: " << dg_formulation);
+    }
+
     db::mutate<db::add_tag_prefix<::Tags::dt, variables_tag>>(
-        box,
-        [dg_formulation, &logical_to_inertial_inverse_jacobian, &mesh,
-         &volume_fluxes](const gsl::not_null<Variables<db::wrap_tags_in<
-                             ::Tags::dt, typename variables_tag::tags_list>>*>
-                             dt_vars_ptr) noexcept {
-          if (dg_formulation == ::dg::Formulation::StrongInertial) {
-            const Variables<tmpl::list<::Tags::div<FluxVariablesTags>...>>
-                div_fluxes = divergence(*volume_fluxes, mesh,
-                                        logical_to_inertial_inverse_jacobian);
-            tmpl::for_each<flux_variables>([&dt_vars_ptr, &div_fluxes](
-                                               auto var_tag_v) noexcept {
-              using var_tag = typename decltype(var_tag_v)::type;
-              auto& dt_var = get<::Tags::dt<var_tag>>(*dt_vars_ptr);
-              const auto& div_flux_var = get<::Tags::div<
-                  ::Tags::Flux<var_tag, tmpl::size_t<Dim>, Frame::Inertial>>>(
-                  div_fluxes);
-              for (size_t storage_index = 0; storage_index < dt_var.size();
-                   ++storage_index) {
-                dt_var[storage_index] -= div_flux_var[storage_index];
-              }
-            });
-          } else if (dg_formulation == ::dg::Formulation::WeakInertial) {
-            ERROR("Weak inertial DG formulation not yet implemented");
-          } else {
-            ERROR("Unsupported DG formulation: " << dg_formulation);
-          }
+        box, [dg_formulation,
+              &div_fluxes](const gsl::not_null<Variables<db::wrap_tags_in<
+                               ::Tags::dt, typename variables_tag::tags_list>>*>
+                               dt_vars_ptr) noexcept {
+          tmpl::for_each<flux_variables>(
+              [&dg_formulation, &dt_vars_ptr,
+               &div_fluxes](auto var_tag_v) noexcept {
+                using var_tag = typename decltype(var_tag_v)::type;
+                auto& dt_var = get<::Tags::dt<var_tag>>(*dt_vars_ptr);
+                const auto& div_flux = get<::Tags::div<
+                    ::Tags::Flux<var_tag, tmpl::size_t<Dim>, Frame::Inertial>>>(
+                    div_fluxes);
+                if (dg_formulation == ::dg::Formulation::StrongInertial) {
+                  for (size_t storage_index = 0; storage_index < dt_var.size();
+                       ++storage_index) {
+                    dt_var[storage_index] -= div_flux[storage_index];
+                  }
+                } else {
+                  for (size_t storage_index = 0; storage_index < dt_var.size();
+                       ++storage_index) {
+                    dt_var[storage_index] += div_flux[storage_index];
+                  }
+                }
```

Comment:
[thinking out lout, not a change request] What are your thoughts on writing this as: 
```
const double formulation_sign = (dg_formulation == ::dg::Formulation::StrongInertial ? -1.0 : 1.0);
for (...) {
  dt_vars[storage_index] += formulation_sign * div_flux[storage_index];
}
```

I'd say that's easier to read for a human, but adds a vector multiply operation (might be free with fma)... I'm not sure what to prefer.

Reply:
Yes, but your code does 2 FLOPs while mine does 1 :)